components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <main>
       <chef-breadcrumbs>
-        <chef-breadcrumb [link]="['/compliance/reports/nodes']">Reports</chef-breadcrumb>
+        <chef-breadcrumb [routerLink]="['/compliance/reports/nodes']" [queryParams]='returnParams'>Reports</chef-breadcrumb>
```

Comment:
Using routerlink, I added the queryParams attribute so that if we have filters, we can apply them dynamically, and if we don't - no harm the link will just go back to the overview page without filters.

Reply:
@SEAjamieD did you test what @msorens mentioned about the [queryParams] not being applied to the page?

Reply:
Yeah - I believe he was referring to using `link` which was applying the query params as comma separated.  So needed to stick with `routerLink`